Syntaxes/Markdown (GitHub Italics).tmLanguage

```diff
@@ -2,12 +2,14 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>fileTypes</key>
+	<array/>
 	<key>hideFromUser</key>
 	<true/>
 	<key>injectionSelector</key>
 	<string>(L:text.html.markdown markup.italic.markdown)</string>
 	<key>name</key>
-	<string>Markdown (GitHub Injected)</string>
+	<string>Markdown (GitHub Italics)</string>
```

Comment:
Hmm, I'm not sure I agree with this rename as it's handling more than just italics. Maybe just `Markdown (GitHub)`?